on_chain/contracts/User.sol

```diff
@@ -2,44 +2,38 @@
 pragma solidity ^0.8.10;
 
 import {TAParserLib} from "../lib/TAParserLib.sol";
-import {console} from "hardhat/console.sol";
+import {Ownable} from "@openzeppelin/contracts/access/Ownable.sol";
 
-contract User {
+contract User is Ownable {
     event AttestedFunctionCallOutput(string output);
 
-    function labeledLog(
-        string memory label,
-        bytes memory data
+    address private enclAttAppPubKeyAddress;
+
+    constructor() Ownable(msg.sender) {
+    }
+
+    function setEnclaveAttestedAppPubKey(
```

Comment:
I decided to go back to the two-step process, because it is easier to explain vis-a-vis the verification process in the documentation